integration/ServiceTestRuleSample/build.gradle

```diff
@@ -26,7 +26,7 @@ allprojects {
 ext {
     buildToolsVersion = "32.0.0"
     coreVersion = "1.5.0"
-    extJUnitVersion = "1.1.4"
-    runnerVersion = "1.5.0"
+    extJUnitVersion = "1.1.5"
+    runnerVersion = "1.5.2"
```

Comment:
Why do we have a minor version jump for the runner version?

Reply:
Y